test/jwks.spec.js

```diff
@@ -187,16 +190,34 @@ t.test('calling the clear cache function resets the cache and clears keys', asyn
   t.end()
 })
 
+t.test('If an issuer provides a domain where there is a missing trailing slash, it should be handled', async t => {
+  t.plan(1)
+  nock('https://localhost/').get('/.well-known/jwks.json').reply(200, jwks)
+  const domainWithMissingTrailingSlash = 'https://localhost'
+  const getJwks = buildGetJwks()
+  const secret = await getJwks.getSecret({ domain: domainWithMissingTrailingSlash, alg: 'RS512', kid: 'KEY' })
+  t.ok(secret)
+  t.end()
+})
+
 const jwks = {
   keys: [
     {
       alg: 'RS512',
       kid: 'KEY',
```

Comment:
why do we have 2 keys with the same ID? are we sure we're testing both?

Reply:
Just the `alg` is different, but I can add more variety of tests

Reply:
my point is, if we're trying to reproduce a realistic scenario, we're not doing it. we can't have two keys with the same id